### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by mboneil10**

### CODE REVIEW

File: cleanslate/doma/templates/home.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends "./base.html" %}
 
-{% block title %}<title>Calendar</title>{% endblock %}
+{% block title %}<title>Home</title>{% endblock %}
```
Comment (L4r1k) [inline cleanslate/doma/templates/home.html:3]: This should already be the case, I changed it to 'Home' in my DRYing PR after Phillip's PR which initialized it to 'Calendar'

**Review by L4r1k — CHANGES_REQUESTED** [review]

File: cleanslate/doma/templates/home.html (hunk 1 above)
Comment (mboneil10) [reply cleanslate/doma/templates/home.html:3]: Errr idk man

Comment (mboneil10) [reply cleanslate/doma/templates/home.html:3]: It's from master. Hold on.

Comment (mboneil10) [reply cleanslate/doma/templates/home.html:3]: Wow. I'm a genius.

**Pull request closed without merging**